lib/internal/http2/core.js

```diff
@@ -32,6 +32,8 @@ const kMaxFrameSize = (2 ** 24) - 1;
 const kMaxInt = (2 ** 32) - 1;
 const kMaxStreams = (2 ** 31) - 1;
 
+const kQuotedString = /^[\x09\x32\x21-\x5b\x5d-\x7e\x80-\xff]*$/;
```

Comment:
oh ha!! thinking hex, typed decimal. sigh

Reply:
I'd still prefer to keep the x80-xff range given the definition of quoted-string. Yes, it *shouldn't* happen, but the spec definition technically allows for it. It will be up to the user to validate that what they get is a valid value.

Reply:
Okay. I can’t agree with that interpretation of the spec but this is not a blocker for me like the corruption issue was.